### PR TITLE
v1 news

### PR DESCRIPTION
**Pull request by isaacabraham**

### CODE REVIEW

File: docs/news.md [hunk 1]
```diff
@@ -1,4 +1,19 @@
...
+We're delighted to announce the release of v1.0 of the SAFE Stack! It's been an exciting year or so for the stack as the stack has evolved and matured. We now have what we feel is a stable and feature-rich stack that achieves the original goals of SAFE, plus a number of optional libraries that people can take advantage of.
+
+With the release of v1, you can now do the following:
+
+* Create a fully working, full-stack web application in F# using the SAFE template in just a few seconds and have it running on your machine from clean in less than a minute.
+* Choose flexible hosting options, including Azure App Service, Google Cloud Platform and Docker.
+* Host with multiple server models including Saturn and Giraffe.
+* Have simultaneous client and server-side debugging sessions in VS Code.
+
+You can upgrade the SAFE Stack template via the dotnet CLI: `dotnet new -i SAFE.Template` to get the latest and greatest version, or `dotnet new -i SAFE.Template::1.0` to pin to v1.0.
```
Comment (theimowski) [inline docs/news.md]: I didn't know about pinning - maybe we should emphasise it more e.g. by placing the information in quick-start?

File: docs/news.md [hunk 2]
```diff
@@ -1,4 +1,19 @@
...
+
+We're looking forward to continuing to evolve the SAFE Stack in the coming months. We have a lot of great ideas for the future, and will be publishing our vNext roadmap in the coming weeks. As always, we're interested in your feedback and ideas, so please reach out to us via the [support](support.md) page.
+
```
Comment (theimowski) [inline docs/news.md]: How about placing a link to the F# eXchange talk?

File: docs/news.md (hunk 1 above)
Comment (isaacabraham) [reply docs/news.md]: Can also do that - happy to do if you like.

Comment (isaacabraham) [reply docs/news.md]: Not sure if that's appropriate for the quick start though?

Comment (theimowski) [reply docs/news.md]: Maybe not quick start, but elsewhere so that it's discoverable

**Pull request closed without merging**